src/test/repository.test.ts

```diff
@@ -1360,3 +1361,73 @@ test('Stopping repository should stop storage provider updates', async (t) => {
   const result = await storageProvider.get(appName);
   t.is(result, undefined);
 });
+
+test('Streaming', async (t) => {
+  t.plan(6);
+  const url = 'irrelevant';
+  const feature = {
+    name: 'feature',
+    enabled: true,
+    strategies: [
+      {
+        name: 'default',
+      },
+    ],
+  };
+  const storageProvider: StorageProvider<ClientFeaturesResponse> = new InMemStorageProvider();
+  const eventSource = {
```

Comment:
in-memory mock is trivial to implement. It's just a thin layer on top of the EventEmitter to adjust the API semantic to SSE API